docs/integrations_sso.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Integration based on single sign on
+parent: Integrations
+has_children: false
+has_toc: false
+nav_order: 5
+---
+
+# Integration based on single sign on
+Use this option for accessing our services by logging in with a supported identity provider.
+
+**Pre-requisite:**
+- You should have an account with a federated identity provider
+- Any interface that allows you to login to this identity provider and make requests to our services.
+
+**Outcome:**
+- You should be able to make web https requests to our services without the need for obtaining credentials from us.
+
+**Our Solution:**
+Matrix federated login
+
+#### Matrix federated login
+**Use it for:** Making web (https) requests to our platform without the need for SmartClean login.
```

Comment:
- Making API calls to our platform without the need for SmartClean login mechanisms such as generation of basic or HMAC credentials per property.
- Integrate Matrix data points and analytics in your existing platform.
- Develop a new application for Matrix ecosystem.

Reply:
Resolved by commit: https://github.com/smartclean/smartclean.github.io/commit/7fb71ebf960323f01d9c713710597ace24e03f07